### ISSUE TITLE
Using data from '3d-pose-baseline'

### ISSUE THREAD

**Issue by shrubb** [rayat137/Pose_3D#2]
Hi again,
In an earlier issue, you wrote that it should be possible to reproduce your results using 2D poses for Human3.6M provided in the [3d-pose-baseline](https://github.com/una-dinosauria/3d-pose-baseline) repository. However, I was only able to achieve 53 mm and 71 mm error using ground truth and stacked hourglass 2D predictions respectively (expected 39.2 mm and 51.9 mm): 
![image](https://user-images.githubusercontent.com/9570420/45044972-539b8580-b07a-11e8-95bd-6dfb0fbc1b5c.png)
This is with default settings (100 epochs, initial learning rate 1e-5 etc.), I only set the dropout rate to 0.5 and changed the data directory. Am I missing something or this is expected?

**Comment by rayat137** [comment]
Hi @shrubb What flags did you use? Did you use the flag --camera_frame? Sorry that I have not finished the documentation yet. I will finish working on it.

**Comment by shrubb** [comment]
No, I didn't. I'll let it train with this flag overnight and check for the improvement tomorrow morning.

Thanks, your effort is awesome!

**Comment by shrubb** [comment]
Woohoo, that worked, thank you again! :tada:
![image](https://user-images.githubusercontent.com/9570420/45078168-f1816580-b0f7-11e8-994a-2d593e111c78.png)

**Comment by rayat137** [comment]
Thanks @shrubb. Glad it has worked so well. In fact the results look better than what I reported in the paper with the 2D poses provided in the 3d-pose-baseline repository.
